cmd/loadtest/main.go

```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/mattermost/mattermost-load-test/config"
```

Comment:
should these still reference the old repo?

Reply:
Good catch, I was fooled cause I had the new branch on the old repository directory and everything worked. I'll update all references.